src/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    internal static class JsonConstants
+    {
+        public const byte OpenBrace = (byte)'{';
+        public const byte CloseBrace = (byte)'}';
+        public const byte OpenBracket = (byte)'[';
+        public const byte CloseBracket = (byte)']';
+        public const byte Space = (byte)' ';
+        public const byte CarriageReturn = (byte)'\r';
+        public const byte LineFeed = (byte)'\n';
+        public const byte Tab = (byte)'\t';
+        public const byte ListSeperator = (byte)',';
+        public const byte KeyValueSeperator = (byte)':';
+        public const byte Quote = (byte)'"';
+        public const byte ReverseSolidus = (byte)'\\';
+        public const byte Solidus = (byte)'/';
+        public const byte BackSpace = (byte)'\b';
+        public const byte FormFeed = (byte)'\f';
+        public const byte Asterisk = (byte)'*';
+
+        public static ReadOnlySpan<byte> TrueValue => new byte[] { (byte)'t', (byte)'r', (byte)'u', (byte)'e' };
+        public static ReadOnlySpan<byte> FalseValue => new byte[] { (byte)'f', (byte)'a', (byte)'l', (byte)'s', (byte)'e' };
+        public static ReadOnlySpan<byte> NullValue => new byte[] { (byte)'n', (byte)'u', (byte)'l', (byte)'l' };
+
+        // Used to search for the end of a number
+        public static ReadOnlySpan<byte> Delimiters => new byte[] { ListSeperator, CloseBrace, CloseBracket, Space, LineFeed, CarriageReturn, Tab, Solidus };
+
+        public static ReadOnlySpan<byte> WhiteSpace => new byte[] { Space, LineFeed, CarriageReturn, Tab };
+
+        public static ReadOnlySpan<byte> EndOfComment => new byte[] { Asterisk, Solidus };
+
+        // Explicitly skipping ReverseSolidus since that is handled separately
+        public static ReadOnlySpan<byte> EscapableChars => new byte[] { Quote, (byte)'n', (byte)'r', (byte)'t', Solidus, (byte)'u', (byte)'b', (byte)'f' };
```

Comment:
Same... can't we just reuse these arrays rather than allocating new ones on every property access?